Userland/Libraries/LibGfx/ICCProfile.cpp

```diff
@@ -194,6 +208,18 @@ ErrorOr<RenderingIntent> parse_rendering_intent(ICCHeader const& header)
     return Error::from_string_literal("ICC::Profile: Invalid rendering intent");
 }
 
+ErrorOr<XYZ> parse_pcs_illuminant(ICCHeader const& header)
+{
+    // ICC v4, 7.2.16 PCS illuminant field
+    XYZ xyz = (XYZ)header.pcs_illuminant;
+
+    /// "The value, when rounded to four decimals, shall be X = 0,9642, Y = 1,0 and Z = 0,8249."
+    if (floor(xyz.x * 10'000) != 9'642 || floor(xyz.y * 10'000) != 10'000 || floor(xyz.z * 10'000) != 8'249)
```

Comment:
The comment says "rounded to" but you `floor()` these values. Shouldn't this be `AK::rounded_to<int>` instead?

Reply:
Oh, yeah!

Reply:
Done, thanks!